projects/core/src/i18n/translation-chunk.service.spec.ts

```diff
@@ -7,7 +7,7 @@ describe('TranslationChunkService', () => {
 
   beforeEach(() => {
     const mockChunksConfig = {
-      key1: 'chunk1',
+      chunk1: ['key1'],
```

Comment:
I think it would be good to add one or two unit tests that will cover the complexity of array config